en/docs/develop/streaming-apps/working-with-the-design-view.md

```diff
@@ -1310,11 +1310,11 @@ In order to define how the Siddhi components in a Siddhi application
 interact with each other to process events, you need to define
 connections between Siddhi components. A connection is defined by
 drawing an arrow from one component to another by dragging the cursor as
-demonstrated below.
+running Siddhi applications
```

Comment:
Is this change required?

Reply:
Changed [0f817e3](https://github.com/wso2/docs-apim/pull/7321/commits/0f817e392b5868be6f6529c83d501fb8e2ac9c48)